lib/bundler/index.rb

```diff
@@ -13,6 +13,9 @@ def self.build
     attr_reader :specs, :all_specs, :sources
     protected :specs, :all_specs
 
+    RUBY = "ruby"
```

Comment:
Yes @segiddins, thanks for the review!

We can do that but only ruby 2.1, 2.2+ will benefit since they're the only rubies that will use the same single shared frozen string:  http://tmm1.net/ruby21-fstrings/

The constant way will work with older and newer rubies AFAIK.